spec/features/user/user_can_bookmark_a_video_spec.rb

```diff
@@ -1,14 +1,25 @@
 require 'rails_helper'
 
 describe 'A registered user' do
+  before :each do 
+    @tutorial_shoes = create(:tutorial, title: "How to Tie Your Shoes")
+    @video1 = create(:video, title: "The Bunny Ears Technique", position: 0, tutorial: @tutorial_shoes)
+
+    @tutorial_pbj= create(:tutorial, title: "How to Make a PBJ")
+    @video2 = create(:video, title: "Is Jam Jelly?", position: 0, tutorial: @tutorial_pbj)
+    @video3 = create(:video, title: "Which knife is the right knife?", position: 1, tutorial: @tutorial_pbj)
+    
```

Comment:
I'd be happy to rename all these videos to something more professional like `title: video1`

Reply:
Every one of these only makes sense when read in Mike Dao's voice